### PR TITLE
Croatian language support

### PR DESCRIPTION
**Pull request by thinkyhead**
Cleanup and filling out of #4048

### CODE REVIEW

**Comment by jbrazio** [comment]
@thinkyhead I had already submitted this with #4052.

**Comment by thinkyhead** [comment]
@jbrazio Sorry, I didn't notice. You might want to incorporate some of the changes I made here that differ from yours.

File: Marlin/language_hr.h [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+#ifndef LANGUAGE_HR_H
+#define LANGUAGE_HR_H
+
+#define MAPPER_NON
+
+// Define SIMULATE_ROMFONT to see what is seen on the character based display defined in Configuration.h
+//#define SIMULATE_ROMFONT
+#define DISPLAY_CHARSET_ISO10646_1 // use the better font on full graphic displays.
+
+#ifndef WELCOME_MSG
```
Comment (ghost) [inline Marlin/language_hr.h]: Well...Is these much #ifndef needed?
I think that It's needed only in language_en.h...

Sample of alteration:
https://github.com/esenapaj/Marlin/commit/3b0c9199985db547536effe38b3a6f6132d21e2b

Comment (thinkyhead) [inline Marlin/language_hr.h]: Aha, how did I miss that?

**Pull request merged**